### DISCUSSION TITLE
"You could have gotten a new PB but your config doesn'a allow it (funbox)"

### DISCUSSION THREAD

**Discussion by Ottaviocr** [monkeytypegame/monkeytype#5461] [Questions] [answered]
What is the meaning of "You could have gotten a new PB but your config doesn't allow it (funbox)"?

What is a PB (I assume it's a new record?)

And what has funbox got to do with it?

**Comment by Miodec** [comment, marked as answer]
Pb is a personal best. Some funboxes alter the word generation and thus are not eligible to count towards your personal best.
